lstchain/tests/test_lstchain.py

```diff
@@ -151,14 +152,20 @@ def test_get_source_dependent_parameters_observed(observed_dl1_files):
 
 
 def test_build_models(simulated_dl1_file, rf_models):
-    from lstchain.reco.dl1_to_dl2 import build_models
     infile = simulated_dl1_file
-
+    custom_config = {
+        "n_training_events": {
+            "gamma_regressors": 0.99,
+            "gamma_classifier_train": 0.78,
+            "gamma_classifier_test": 0.21,
+            "proton_classifier": 0.98
```

Comment:
gamma_classifier_test seems to me a bit of a confusing name.  This is actually the fraction used in the _training_ of the classifier, right? (it is "test" only from the point of view of the tmp_regressors). 

Also, if we are not using the full gamma statistics, we could use the same training sample for the tmp regressors and the final ones. In that case, wouldn't it be enough, e.g.:

"gamma_regressors": 0.70
"gamma_classifier": 0.30     (could default to 1 -  fraction_gamma_regressors)
"proton_classifier": 1.00

Reply:
Hi, this comment about the options naming still holds.  My "forget my previous comment" refers to another one that I deleted (it was about the test with fractions adding to more than 1, to make it fail)